lightning/src/ln/wire.rs

```diff
@@ -16,6 +16,13 @@
 use ln::msgs;
 use util::ser::{Readable, Writeable, Writer};
 
+/// Maximum Lightning message data length according to
+/// [BOLT-8](https://github.com/lightningnetwork/lightning-rfc/blob/v1.0/08-transport.md#lightning-message-specification):
+/// "The maximum size of any Lightning message MUST NOT exceed 65535 bytes.
```

Comment:
Not sure the quote adds much value, but it takes readers' attention.
Keeping a number is also not useful, because it's already in the field.

Reply:
You mean quotation marks or the quoted text?

Reply:
I mean the text. I think it's enough to say:
" /// Maximum Lightning message data length according to BOLT-8"

Reply:
I've used spec referencing model from c-lightning, where they directly cite a part of the spec right in the code

Reply:
Never read c-lightning, but big files generally want me to close the tab all the time. And this seems like a path to get there. Maybe you are right.

Anyway, I don't have the energy to argue about this, feel free to do whatever :)

Reply:
The counter argument is that the spec changes, so quoting it can lead to the code comment getting out of sync with the spec as it evolves. It is also mentioned in [BOLT 1](https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#lightning-message-format):

> The size of the message is required by the transport layer to fit into a 2-byte unsigned int; therefore, the maximum possible size is 65535 bytes.


My preference would be to reference the spec but leave out the quote.

Reply:
In general, we tend to never quote the spec - the spec can be confusing, and docs should be self-contained - users should be able to read them and figure out what's going on without having to flip to another reference.

Reply:
Done